keel-core/src/main/kotlin/com/netflix/spinnaker/keel/pause/ActuationPauser.kt

```diff
@@ -66,9 +67,9 @@ class ActuationPauser(
   fun resourceIsPaused(id: String): Boolean =
     pausedRepository.resourcePaused(id)
 
-  fun pauseApplication(application: String) {
+  fun pauseApplication(application: String, user: String) {
```

Comment:
Functions that pause things now receive a user as an argument, so we can record who paused them in the database.